lib/content-services/src/lib/version-manager/version-list.component.scss

```diff
@@ -1,10 +1,19 @@
 .adf-version-list {
-    &-viewport {
+    .adf-version-list-viewport {
```

Comment:
Generally I wonder if this is the right approach, in every other place we rather use ellipsis and title attribute to handle such cases and I think we should go with the same approach here, I would leave the current ellipsis as reported in the ticket and added a title with tooltip if it's not there already, what do you think @AleksanderSklorz

Reply:
I discussed that with @akashrathod28 before work on that task and he wanted solution with wrapping text. I can go with any of these solutions, just please let me know @akashrathod28 @MichalKinas which one should I take.

Reply:
@akashrathod28 we can discuss this with Agnieszka but wrapping the text in this case would be inconsistent with other places that we have in application

Reply:
@AleksanderSklorz @akashrathod28 I prefer consistent solutions in the app, so I give my vote to ellipsis and tooltip as @MichalKinas suggested.

Reply:
thank you @AgnieszkaOpilska for answer, I will go with tooltip ald ellipsis approach

Reply:
I discussed with @AleksanderSklorz  we will go with tooltip approach.

Reply:
done, I've pushed correction